.github/actions/inductor-xpu-e2e-test/action.yml

```diff
@@ -216,7 +229,7 @@ runs:
           mode=$(echo ${{ inputs.mode }} |sed 's/,/ /g')
           python inductor_perf_summary.py -p ${dt} -s ${suite} -m ${mode}
         done
-        cd ${{ github.workspace }} && rm -rf inductor_log && cp -r ../pytorch/inductor_log .
+        cd ${{ github.workspace }} && cp -r ../pytorch/inductor_log .
```

Comment:
why we need this change?

Reply:
If we remove inductor_log here, it will remove accuracy summary data that is generated from previous stage.